app/uk/gov/hmrc/exports/models/declaration/ExportItem.scala

```diff
@@ -63,7 +63,12 @@ object TaricCode {
   implicit val format: OFormat[TaricCode] = Json.format[TaricCode]
 }
 
-case class ItemType(nationalAdditionalCode: Seq[String], statisticalValue: String)
+case class NactCode(nactCode: String)
```

Comment:
Let use full name would be less confusing for new people

Reply:
Its all confusing.  Whats a "TARIC" or a CUS